packages/lesswrong/components/dropdowns/NotifyMeToggleDropdownItem.tsx

```diff
@@ -44,15 +50,24 @@ export const NotifyMeToggleDropdownItemInternal = ({
   const {DropdownItem, ToggleSwitch} = Components;
 
   const afterIcon = useCallback(
-    () => <ToggleSwitch value={subscribed} className={classes.toggle} smallVersion />,
-    [subscribed, ToggleSwitch, classes.toggle],
+    () => {
+      if (useCheckboxIcon) {
+        return <Checkbox checked={subscribed} />
+      } 
+
+      return <ToggleSwitch value={subscribed} className={classes.toggle} smallVersion />
+    },
+    [subscribed, useCheckboxIcon, ToggleSwitch, classes.toggle ],
   );
 
   return (
     <DropdownItem
       title={title}
       onClick={toggleSubscribed}
       afterIcon={afterIcon}
+      loading={loading}
```

Comment:
This could be the loading spinner change

Reply:
Ah, makes sense. So you say above "getting a flash of loading spinner" – do you want me to forum gate changes that cause this?

Reply:
I think so? I haven't thought about what's going on too deeply, it could be that the loading spinner is "correct."